### DISCUSSION TITLE
NPC Recipes (what they can make)

### DISCUSSION THREAD

**Discussion by hotluck** [ldtteam/minecolonies#8290] [Ideas]
Using common sense, you can figure out some recipes and which npc can make them, but there are others that I can't fathom out and can find no details when searching, so my proposal is this -

If you try and add a recipe to an NPC (e.g. lantern to the blacksmith), instead of saying that either the recipes allowed is full or cannot be made by that person, they could say which person can actually make that item? Is that possible? If not, could a list of items (ones specifically created by minecolonies & domum ornamentum, and also ones that aren't totally using blocks specific to an npc, such as not all stone for stonemason) be added to the wiki?

I hope that makes sense lol.

**Comment by uecasm** [comment]
Just look in JEI, it's what it's there for.

**Reply by hotluck** [reply]
Apologies for the delayed response but I've just started again and yes JEI works for things like polished diorite, but not things like polished diorite panels. I tried the stonemason, but it says they don't do that customer recipe, so who does?

**Reply by uecasm** [reply]
Right-click on the ingredient block (polished diorite in this case), or hover and press U, then check the tabs:
<img width="407" height="406" alt="image" src="https://github.com/user-attachments/assets/ed2f3965-a17c-4ded-8d67-673fe42f89fe" />

This shows that the stonemason does make DO blocks including panels using that block.

If you don't see the same, then you may need to check your logs to see if any tags are failing to load; it may be a bug in your modpack or datapacks.  Or you may be on an outdated version.
